### DISCUSSION TITLE
Feature: add speed (playback rate) control/selector

### DISCUSSION THREAD

**Discussion by aminamos** [muxinc/elements#476] [Ideas]
Allow users to set their desired playback rate, provided a set number of options like:

- 0.25
- 0.5
- 0.75
- 1.0
- 1.25
- 1.5
- 1.75
- 2

**Comment by gkatsev** [comment]
This is available now! You can set `playbackrates` attribute as a space-separate list like so:
```html
<mux-player playbackrates="1 2 3"></mux-player>
```
